expression/builtin_miscellaneous_test.go

```diff
@@ -0,0 +1,13 @@
+// Copyright 2017 PingCAP, Inc.
```

Comment:
Is builtin function sleep moved from other place? Does it has test? if so, please also move test code.